ci-operator/config/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-master.yaml

```diff
@@ -23,14 +23,13 @@ tests:
   steps:
     cluster_profile: packet
     workflow: baremetalds-e2e
-- as: e2e-metal-ipi-dualstack
+- as: e2e-metal-ipi-ovn-dualstack
   steps:
     cluster_profile: packet
-    pre:
-    - ref: baremetalds-packet-setup
-    - ref: ipi-install-rbac
-    - ref: baremetalds-devscripts-conf-dualstack
-    - ref: baremetalds-devscripts-setup
+    env:
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4v6
+        NETWORK_TYPE=OVNKubernetes
```

Comment:
Is this a new requirement? I did not see it before in `baremetalds-devscripts-conf-dualstack`

Reply:
It’s implied in dev-scripts by the IP_STACK but I wanted to be explicit in case OpenShiftSDN supports dual stack or IPv6